src/lib/builtins.pl

```diff
@@ -33,6 +33,13 @@
 internal settings and basic I/O are all here.
 */
 
+%% (:)(+M, +T)
```

Comment:
This is invalid syntax. Where did this originate?

Reply:
I based it on "Coding Guidelines for Prolog" which has no examples of symbolic operators. I think I'm confused with Haskell, which uses parentheses to turn an infix operator like `+` into a prefix function.

Reply:
I am completely baffled by the 👎 from @UWN.

I thought the "invalid syntax" was about the parentheses around the `:` token, which I've since removed. Could you please ELI5 or suggest a correction?